modules/virt-configuring-rate-limiters.adoc

```diff
@@ -0,0 +1,19 @@
+
+// Module included in the following assemblies:
+//
+// * virt/advanced_vm_management/virt-vm-control-plane-tuning.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-rate-limiters_{context}"]
+= Configuring rate limiters
+
+To compensate for large-scale burst rates, scale the `QPS` (Queries per Second) and `burst` rate limits to process a higher rate of client requests or API calls concurrently for each component.
+
+.Procedure
+
+* Apply a `jsonpatch` annotation to adjust the `kubevirt-hyperconverged` cluster configuration by using `tuningPolicy` to apply scalable tuning parameters. This tuning policy automatically adjusts all virtualization components (`webhook`, `api`, `controller`, `handler`) to match the `QPS` and `burst` values specified by the profile.
+
+[IMPORTANT]
+====
+It is recommended to open a Support Exception before applying this method of tuning.
```

Comment:
@jeniferh do customers still require a support exception?

Reply:
AFAIK any hco change still requires a SE, cc @fabiand to be sure

Reply:
@fabiand Can you confirm?

Reply:
@dominikholler I was unclear at the start that the support exception referred to the jsonpatch, which is the entire procedure in this module. We don't document unsupported features in the CNV docs unless they are TP or DP, and only if clearly marked. I chatted with @djfjeff today about this and I'm removing this module from the docs until it is supported.